### ISSUE TITLE
Button trigger build for Pull requests.

### ISSUE THREAD

**Issue by sashanas** [tomasbjerre/pull-request-notifier-for-bitbucket#301]
Hi Tomas!  Thank you again for the plugin.  It's really wonderful!  I had a quick question and I am not sure if what I am trying to do with the plugin is possible.  I can easily set up a button and the build associated with it, which works just fine.  I was wondering if there is any way to configure a button so I can build only PR# from which the button is triggered.  As an example: let's say I open PR 123 and I select a trigger PR build, it would only trigger build for PR 123 in jenkins.  Right now my button is only targetting static build of the branch and URL of the trigger doesn't change.  I was wondering if there is any way to configure for the job URL to take value from the PR that triggered it?  

Thanks in advance for all your help.

Current button setup:

![screen shot 2018-09-10 at 3 50 33 pm](https://user-images.githubusercontent.com/40475438/45320689-5a5e4880-b511-11e8-8264-b769eb07bc6b.png)
![screen shot 2018-09-10 at 3 49 24 pm](https://user-images.githubusercontent.com/40475438/45320690-5a5e4880-b511-11e8-82ea-a110e1821783.png)

**Comment by tomasbjerre** [comment]
Yes there are plenty of variables to use, check the [readme](https://github.com/tomasbjerre/pull-request-notifier-for-bitbucket/blob/master/README.md).

**Comment by sashanas** [comment]
Thank you Tomas!  Was able to get it working!
